### PR TITLE
fix(docker): switch to yarn

### LINKED ISSUE #933: Browserforce fails to load
**Describe the bug**

In the latest docker image (https://github.com/orgs/dxatscale/packages/container/sfpowerscripts/18755802?tag=13.0.11-2146811612) for April 22, browserforce fails to load with

```
ERROR running browserforce:apply: Failed to launch the browser process!

/sfdx_plugins/.local/share/sfdx/node_modules/puppeteer/.local-chromium/linux-901912/chrome-linux/chrome: error while loading shared libraries: libxshmfence.so.1: cannot open shared object file: No such file or directory

```

**To Reproduce**
1. Install a package with browserforce as a pre/post step

**Expected behavior**

It should be executed successfully

**Screenshots**

**Platform Details (please complete the following information):**
 - OS: ubuntu
 - Version [e.g. CLI Version  eg: 1.6.6] : 13.0.11-2146811612
 - Salesforce CLI(sfdx cli) Version: 7.145.0
 - CI Platform: all

**Additional context**

### LINKED ISSUE #936: Sudo command is missing in the latest version of docker container for sfpowerscripts
**Describe the bug**
Running scripts using the sudo command shows (sudo: command not found).

**To Reproduce**
Run any script prefaced by sudo will cause it to fail with error sudo: command not found.

**Expected behavior**
The container would run the script prefaced by sudo.

**Screenshots**
image
**Platform Details (please complete the following information):**
 - OS: Ubuntu 20.04
 - Docker Version: [release-apr22,latest]
 - CI Platform: Azure Pipelines

**Additional context**
Workaround is to use previous version of the docker container as the issue is not present there.

### PR DESCRIPTION
**Pull request by azlam-abdulsalam**
#### Description

-  switch to yarn for installing sfdx-cli, 
-  install pupetteer globally on v10 to meet browserforce
-  ensure all chrome dependencies are met 
-  install sudo and other shared libs as in previous docker images

fix #936 fix #933

#### Checklist
All items have to be completed before a PR is merged

- [x] Adhere to [Contribution Guidelines](https://docs.dxatscale.io/about-us/contributing-to-dx-scale)
- [x] Updates to Decision Records considered?
- [x] Updates to documentation at [DX@Scale Guide](https://github.com/dxatscale/dxatscale-guide) considered?
- [x] Tested changes?
- [x] Unit Tests new and existing passing locally?

### CODE REVIEW

File: dockerfiles/sfpowerscripts.Dockerfile [hunk 1]
```diff
@@ -65,23 +70,27 @@ RUN export XDG_DATA_HOME && \
...
-
-
-ENV SFDX_CONTAINER_MODE true
+#Install Puppeteer
+RUN yarn global add puppeteer@10.4.0
 
+# Install all shared dependencies for chrome and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+# installs, work.
+RUN apt-get update && apt-get install -qq gconf-service libappindicator1 libasound2 libatk1.0-0 \
```
Comment (aly76) [inline dockerfiles/sfpowerscripts.Dockerfile:91]: Should cleanup apt cache after installation to keep image size small

File: dockerfiles/sfpowerscripts.Dockerfile [hunk 2]
```diff
@@ -90,9 +99,16 @@ RUN echo 'y' | sfdx plugins:install sfdmu@4.13.0
 RUN echo 'y' | sfdx plugins:install sfpowerkit@4.2.5
 
 
+# install sfpowerscripts
 RUN echo 'y' | sfdx plugins:install @dxatscale/sfpowerscripts@$SFPOWERSCRIPTS_VERSION
 
 
+# Install PMD
```
Comment (aly76) [inline dockerfiles/sfpowerscripts.Dockerfile]: Reorder, as PMD version does not change most frequently

File: dockerfiles/sfpowerscripts.Dockerfile (hunk 1 above)
Comment (aly76) [reply dockerfiles/sfpowerscripts.Dockerfile:91]: Should reorder as the dependencies don't change often

**Review by aly76 — DISMISSED** [review]

File: dockerfiles/sfpowerscripts.Dockerfile (hunk 2 above)
Comment (azlam-abdulsalam) [reply dockerfiles/sfpowerscripts.Dockerfile]: not sure whether it matters, as we are not using cache anymore

File: dockerfiles/sfpowerscripts.Dockerfile (hunk 1 above)
Comment (azlam-abdulsalam) [reply dockerfiles/sfpowerscripts.Dockerfile:91]: had issues with pupetteer not recognizing these libs if installed earlier,  will need to investigate further

**Pull request merged**